test/testother.cpp

```diff
@@ -8199,10 +8195,9 @@ class TestOther : public TestFixture {
         }
 
         {
-            Settings s = settingsBuilder().checkUnusedTemplates().build();
```

Comment:
Unnecessary as `checkUnusedTemplates` is `true` by default.

Reply:
This now uses `_settings` which has `std.cfg` loaded but I do think this is not an issue.